Comparison Engine/Comparison Engine/Form1.cs

```diff
@@ -163,9 +163,9 @@ private void openChildFormDrink(Drink drink)
             configureChildForm(profileForm);
         }
 
-        private void openChildFormUserContribution(Drink drink, Bar bar)
+        private void openChildFormUserContribution()
```

Comment:
Should probably capitalize the first letter of methods (since that's a requirement apparently). Some of the methods handling buttons also don't start with a capital letter